useful-scripts/sql/FootnoteData.sql

```diff
@@ -97,21 +97,23 @@ END CATCH
 GO;
 
 
-DELETE FROM SubjectFootnote WHERE 1=1;
-DELETE FROM IndicatorFootnote WHERE 1=1;
-DELETE FROM FilterItemFootnote WHERE 1=1;
-DELETE FROM FilterGroupFootnote WHERE 1=1;
-DELETE FROM FilterFootnote WHERE 1=1;
-DELETE FROM Footnote WHERE 1=1;
+-- DELETE FROM SubjectFootnote WHERE 1=1;
+-- DELETE FROM IndicatorFootnote WHERE 1=1;
+-- DELETE FROM FilterItemFootnote WHERE 1=1;
+-- DELETE FROM FilterGroupFootnote WHERE 1=1;
+-- DELETE FROM FilterFootnote WHERE 1=1;
+-- DELETE FROM Footnote WHERE 1=1;
 
 
 --
 -- Footnotes
 --
+
+-- Absence footnotes
 DECLARE @footnote_id_1 uniqueidentifier = 'fbb6262f-213a-453a-98ca-b832d6ae1c16';
 DECLARE @footnote_id_2 uniqueidentifier = 'e9076bab-6ff7-4c92-8972-2fb4affbe977';
 DECLARE @footnote_id_3 uniqueidentifier = '128c6153-4937-40da-a99f-e137fc8c41e5';
-DECLARE @footnote_id_4 uniqueidentifier = '41c1eb5a-8415-45eb-bc49-cfa42382ebba';
+-- footnote_id_4 removed as part of BAU-424
```

Comment:
I would have just removed this and renumbered them all. It's a hangover from when we used integer ids' rather than guids. The numbers now mean nothing.

Reply:
I've renumbered them all. I just left the comment because I didn't want to renumber them all and thought someone might wonder where #4 is missing.